pyradiosky/skymodel.py

```diff
@@ -499,33 +503,43 @@ def _calc_average_rotation_matrix(self):
 
         return R_really_orthogonal
 
-    def _calc_rotation_matrix(self):
+    def _calc_rotation_matrix(self, inds=None):
         """
         Calculate the true rotation matrix from RA/Dec to AltAz for each component.
 
+        Parameters
+        ----------
+        inds: array_like, optional
+            Index array to select components.
+            Defaults to all components.
+
         Returns
         -------
         array of floats
             Rotation matrix that defines the mapping (RA,Dec) <--> (Alt,Az),
             shape (3, 3, Ncomponents).
         """
+        if inds is None:
+            inds = range(self.Ncomponents)
```

Comment:
Looks like this line isn't covered by tests.

Reply:
The latest commit should cover this line now.